go-controller/pkg/ovn/controller/admin_network_policy/admin_network_policy.go

```diff
@@ -32,6 +49,888 @@ func (c *Controller) processNextANPWorkItem(wg *sync.WaitGroup) bool {
 	return true
 }
 
+// NOTE: Iteration v1 of ANP will only support upto 100 ANPs
+// We will use priority range from 30000 (0) to 20000 (99) ACLs (both inclusive, note that these ACLs will be in tier1)
+// In order to support more in the future, we will need to fix priority range in OVS
+// See https://bugzilla.redhat.com/show_bug.cgi?id=2175752 for more details.
+const (
+	ANPFlowStartPriority            = 30000
+	ANPMaxRulesPerObject            = 100
+	ANPIngressPrefix                = "ANPIngress"
+	ANPEgressPrefix                 = "ANPEgress"
+	ANPExternalIDKey                = "AdminNetworkPolicy" // key set on port-groups to identify which ANP it belongs to
+	OVNKSupportedPriorityUpperBound = 99                   // corresponds to 20100 ACL priority
```

Comment:
some of these have to be for the unit tests which are outside in the `ovn` package.. see https://github.com/ovn-org/ovn-kubernetes/pull/3659#discussion_r1267837168

Reply:
The `admin_network_policy` package you mean?

Reply:
so the `admin_network_policy_test.go` and `baseline_admin_network_policy_test.go` unit tests are unfortunately outside in `ovn` package like the `egress_service_test.go`. These files rely on `fakeOVN` and `defaultNetworkController` for testing things that they can't be within the `admin_network_policy` package. The reason why we need the full integration tests is because there are dependencies on when a pod is added and its update arrives versus the anp add, so testing just the anpController alone without namespace/pod add/delete events seemed _not enough_

Thus I needed to export these variables since they are used from the unit tests. but `ovnkSupportedPriorityUpperBound` is not used outside the package so I can fix this one to not be exported.

Reply:
Ah, I see.

It looks to me though that this level driven controllers should be way easier to test than depending on `fakeOVN` or the full test harness suite we depended on for the default network controller. The predicates that test whether an ANP should be enqueued or not can be tested by themselves. And for the main controller we should only need the fake kube client and libovsdb test harness.

If that's not enough, I guess the other course of action is take all the test harness needed out of the `ovn` package into the `testing` package so that it can be easily shared among all tests.

Reply:
```
If that's not enough, I guess the other course of action is take all the test harness needed out of the ovn package into the testing package so that it can be easily shared among all tests.
```
yes this is the ideal cleanup that I would like to do... its on my TODO list because this can also help clean stuff for other features.